sdks/python/container/Dockerfile

```diff
@@ -19,8 +19,64 @@
 FROM python:2
 MAINTAINER "Apache Beam <dev@beam.apache.org>"
 
-# TODO(herohde): preinstall various packages for better startup
-# performance and reliability.
+# Install native bindings required for dependencies.
+RUN apt-get update && \
+    apt-get install -y \
+       # These packages are needed for "pip install python-snappy" below.
+       libsnappy-dev \
+       # This package is needed for "pip install pyyaml" below to have c bindings.
+       libyaml-dev
```

Comment:
You should remove the apt-get package lists as part of this command: rm -rf /var/lib/apt/lists/*

Reply:
Done.